OsmAnd/src/net/osmand/plus/profiles/LocationIcon.java

```diff
@@ -18,12 +19,12 @@
 
 public enum LocationIcon {
 
-	STATIC_DEFAULT(R.drawable.map_location_default, R.drawable.map_location_default_view_angle, "model_map_default_location"),
-	STATIC_CAR(R.drawable.map_location_car, R.drawable.map_location_car_view_angle, "model_map_car_location"),
-	STATIC_BICYCLE(R.drawable.map_location_bicycle, R.drawable.map_location_bicycle_view_angle, "model_map_bicycle_location"),
-	MOVEMENT_DEFAULT(R.drawable.map_navigation_default, "model_map_car_bearing"),
-	MOVEMENT_NAUTICAL(R.drawable.map_navigation_nautical, "model_map_navigation_nautical"),
-	MOVEMENT_CAR(R.drawable.map_navigation_car, "model_map_navigation_car"),
+	STATIC_DEFAULT(OsmAndAppCustomizationFields.STATIC_DEFAULT, OsmAndAppCustomizationFields.STATIC_DEFAULT_VIEW_ANGLE, "model_map_default_location"),
```

Comment:
Doesn't sound reasonable as it static on static, so we need to keep old constants and allow customization via model